app/routes/g_.$groupId.e.new.tsx

```diff
@@ -0,0 +1,77 @@
+import type { FormEvent } from 'react';
```

Comment:
Let's keep the routing naming convention consistent. So either `g -> groups` and `e -> events` or all existing `groups ->`.